src/features/comentsSlice.ts

```diff
@@ -0,0 +1,72 @@
+import { createAsyncThunk, createSlice } from '@reduxjs/toolkit';
```

Comment:
The file name and import paths contain a typo: 'comentsSlice'. It should be corrected to 'commentsSlice' to match the intended functionality.